src/com/pentalog/pentastagiu/homework2/shop/Basket.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Basket {
+    private Map<Product, Integer> basket;
```

Comment:
e putin neclar sa ai in clasa Basket un field basket, mai clar ar fi productToQuantity sau productsByQuantity, mi-am dat seama de asta cand am citit codul din Customer si nu am inteles la ce anume se refera asa ca am revenit aici sa vad despre ce e vorba